src/it/java/net/spy/memcached/util/ConnectionUtilLinuxImpl.java

```diff
@@ -31,8 +31,14 @@
 	 */
 	@Override
 	public void addLocalMemcachedServer(int...ports) throws Exception {
-		for(int p: ports) {
-			addLocalMemcachedServer(TestConfig.MEMCACHED_PATH, p, getMemcachedParams());		  
+		if (TestConfig.isTlsMode()){
+			for(int p: ports) {
+				addLocalMemcachedServer(TestConfig.MEMCACHED_PATH, p, getMemcachedTlsParams());		  
+			}
+		} else {
+			for(int p: ports) {
+				addLocalMemcachedServer(TestConfig.MEMCACHED_PATH, p, getMemcachedParams());	
+			}
```

Comment:
I suggest you leave this code unchanged, and simply change the method `getMemcachedParams()` to handle the TLS parameters if TLS is enabled.

Reply:
Will do